users/models.py

```diff
@@ -0,0 +1,54 @@
+"""
+Holds database models
+"""
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from .managers import UserManager
+
+# pylint: disable = no-member
+
+
+class TimeStamp(models.Model):
+    """
+    TimeStamp model to check when the object was created and modified
+    """
+
+    created_on = models.DateTimeField(auto_now_add=True, editable=False)
+    updated_on = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        """
+        Tells that this class is abstract
+        """
+
+        abstract = True
+
+
+class User(AbstractBaseUser, PermissionsMixin, TimeStamp):
```

Comment:
There are no migrations created for this model. You should run `./manage.py makemigrations users` to create the migrations file to create the proper user table in the database. And that migrations, in fact all the migrations should be pushed to VCS and not added in .gitignore.

Reply:
I have removed migrations from `.gitignore`. They are added to VCS now.